lib/classes/Service.test.js

```diff
@@ -763,6 +763,141 @@ describe('Service', () => {
         expect(consoleLogStub.callCount).to.equal(2);
       });
     });
+
+    describe('stage name validation', () => {
+      function simulateRun(serverless) {
```

Comment:
Note: These steps are tricky but I believe it is required to populate function names.